resources/views/robots.blade.php

```diff
@@ -0,0 +1,8 @@
+<?php header('Content-Type: text/plain; charset=utf-8'); ?>
+Sitemap: {{ env('APP_URL') }}/sitemap.xml
+
+User-Agent: *
+Disallow: /
```

Comment:
oh, I would like to get some suggestions around which pages we should disallow from the user-agents; I am thinking disallow every pages those are not /about/, but for now I left it as what it was before.

Reply:
Probably anything that would give a 400 error should be blocked from crawling but the rest are okay. Actually I don't think it could reach those pages that are behind the login anyways. I guess explicitly disallow the following:

- admin/
- api/
- status/

Perhaps @greatislander would have some thoughts on this.